lib/utils/HttpHelper.dart

```diff
@@ -172,6 +179,11 @@ class HttpHelper {
   Map<String, String>? _buildHeaders({
     User? user,
     bool isTestModeActive = false,
+
+    /// Please only use for Robotoff,
```

Comment:
Please move the comment upwards, before the method.
Actually as you repeating yourself, I would put the bulk of the comment only once (here), and just put a `/// Please only use for Robotoff, see [_buildHeaders] for details`